docs/docs/installation.md

```diff
@@ -36,18 +44,11 @@ At least one [SDP solver](https://yalmip.github.io/allsolvers/), such as [SeDuMi
 The proper functioning of YALMIP can be checked with the command `yalmiptest`.
 
 If you do not wish to install these tools on you own, you can alternatively use the YALMIP and SDPT3 submodules that are included in **RepLAB** for testing purposes. 
-For this, update all submodules by issuing the command
-
-```
-git submodule update
-```
-
-from within the replab folder. 
 
-Within MATLAB/Octave, use then the following commands to finish the installation of the solver:
+Within MATLAB/Octave, use the following commands to finish the installation of the solver:
 
 ```
-replab_addpaths(1)
+replab_addpaths(2,1)
```

Comment:
I guess this correction is needed to use convex optimization support (two arguments now).

Reply:
1 would be the default, but 2 is also ok :+1: